packages/babel-plugin-transform-es2015-arrow-functions/test/fixtures/arrow-functions/super-in-class-method/expected.js

```diff
@@ -0,0 +1,26 @@
+class SuperClass {
+  doStuff() {
+    return "superclass";
+  }
+}
+
+class MyClass extends SuperClass {
+  superDotAccess() {
+    var _super = super;
```

Comment:
Argh, you're right. I swore I ran this code in the console, but I must have been mistaken, because it's giving me errors now.

Another option for dot access would be:

```
class MyClass extends SuperClass {
  superDotAccess() {
    var _super_doStuff = super.doStuff;

    const arrow = function () {
      return _super_doStuff();
    };
    return arrow() + ", but in MyClass";
  }
}
```

but that wouldn't work for bracket access, of course. Another thought would be adding a _superCall() method to the class or object literal:

```
class MyClass extends SuperClass {
  superDotAccess() {
    const arrow = function () {
      return _superCall("doStuff");
    };
    return arrow() + ", but in MyClass";
  }

  _superCall(fnName, ...args) {
    super[fnName](...args);
  }
}
```

I think this could work for both bracket and dot access, though it would of course not handle `super()`.